internal/utils/config.go

```diff
@@ -347,6 +349,17 @@ type (
 		TestOTP             map[string]string `toml:"test_otp"`
 	}
 
+	hook struct {
+		MFAVerificationAttempt      extensibilityPointConfig `toml:"mfa_verification_attempt" mapstructure:"mfa_verification_attempt"`
+		PasswordVerificationAttempt extensibilityPointConfig `toml:"password_verification_attempt" mapstructure:"password_verification_attempt"`
+		CustomAccessToken           extensibilityPointConfig `toml:"custom_access_token" mapstructure:"custom_access_token"`
```

Comment:
```suggestion
		MFAVerificationAttempt      extensibilityPointConfig `toml:"mfa_verification_attempt"`
		PasswordVerificationAttempt extensibilityPointConfig `toml:"password_verification_attempt"`
		CustomAccessToken           extensibilityPointConfig `toml:"custom_access_token"`
```

No need to use mapstructure. See comment here https://github.com/supabase/cli/blob/main/internal/utils/config.go#L230